src/ostorlab/assets/api_schema.py

```diff
@@ -0,0 +1,39 @@
+"""API Schema asset."""
+
+import dataclasses
+from typing import Optional
+
+from ostorlab.assets import asset
+
+PROTO_FIELD = "api_schema"
```

Comment:
If PROTO_FIELD is used only once, I might consider directly using the string "api_schema" in the property method. but If the constant might be used in other parts of the module in the future, keeping it as a module-level constant is beneficial.